### PR TITLE
apm: Stub IsCpuOverclockEnabled

### PR DESCRIPTION
**Pull request by german77**
Stub IsCpuOverclockEnabled. Used by MONSTER HUNTER RISE DEMO

### CODE REVIEW

File: src/core/hle/service/apm/interface.cpp [hunk 1]
```diff
@@ -78,6 +78,14 @@ void APM::GetPerformanceMode(Kernel::HLERequestContext& ctx) {
     rb.PushEnum(controller.GetCurrentPerformanceMode());
 }
 
+void APM::IsCpuOverclockEnabled(Kernel::HLERequestContext& ctx) {
+    LOG_DEBUG(Service_APM, "(STUBBED) called");
```
Comment (Morph1984) [inline src/core/hle/service/apm/interface.cpp:82]:
```suggestion
    LOG_WARNING(Service_APM, "(STUBBED) called");
```

**Comment by FearlessTobi** [comment]
Fixes #5316.

**Review by bunnei — APPROVED** [review]

**Comment by fuhuan416** [comment]
Tested with EA-1322, still a black screen, I waited for about 10 minutes, the log file should say that it is complete, here is the log file :)
[yuzu_log.zip](https://github.com/yuzu-emu/yuzu/files/5825699/yuzu_log.zip)

**Comment by fuhuan416** [comment]
> Tested with EA-1322, still a black screen, I waited for about 10 minutes, the log file should say that it is complete, here is the log file :)
> [yuzu_log.zip](https://github.com/yuzu-emu/yuzu/files/5825699/yuzu_log.zip)

These entries were added to the log file when I exited the simulator:

[ 161.184185] Service.NVDRV <Warning> core\hle\service\nvdrv\devices\nvmap.cpp:IocParam:198: (STUBBED) called type=1
[ 165.541117] Service.NVDRV <Warning> core\hle\service\nvdrv\devices\nvmap.cpp:IocParam:198: (STUBBED) called type=1
[ 166.757602] Service.NVDRV <Warning> core\hle\service\nvdrv\devices\nvmap.cpp:IocParam:198: (STUBBED) called type=1
[ 171.873641] Service.NVDRV <Warning> core\hle\service\nvdrv\devices\nvmap.cpp:IocParam:198: (STUBBED) called type=1
[ 172.839601] Service.NVDRV <Warning> core\hle\service\nvdrv\devices\nvmap.cpp:IocParam:198: (STUBBED) called type=1
[ 173.328567] Service.NVDRV <Warning> core\hle\service\nvdrv\devices\nvmap.cpp:IocParam:198: (STUBBED) called type=1
[ 173.549509] Service.NVDRV <Warning> core\hle\service\nvdrv\devices\nvmap.cpp:IocParam:198: (STUBBED) called type=1
[ 175.750030] Service.NVDRV <Warning> core\hle\service\nvdrv\devices\nvmap.cpp:IocParam:198: (STUBBED) called type=1
[ 177.390807] Service.NVDRV <Warning> core\hle\service\nvdrv\devices\nvmap.cpp:IocParam:198: (STUBBED) called type=1
[ 177.400951] Service.NVDRV <Warning> core\hle\service\nvdrv\devices\nvmap.cpp:IocParam:198: (STUBBED) called type=1

**Comment by fuhuan416** [comment]
EA-1384
OpenGL, FPS is normal at 30 frames, the emulator is automatically closed when loading, this is the log file:
[yuzu_log - MMRiseDemo.OpenGL.zip](https://github.com/yuzu-emu/yuzu/files/5879789/yuzu_log.-.MMRiseDemo.OpenGL.zip)

Vulkan, FPS is very low, 3~5 frames, the simulator will automatically shut down when loading, this is the log file:
[yuzu_log - MMRiseDemo.Vulkan.zip](https://github.com/yuzu-emu/yuzu/files/5879790/yuzu_log.-.MMRiseDemo.Vulkan.zip)

**Pull request merged**